helperFunctions/subscribeHelpers/InitiateSalesforceSessionHelpers/CheckChatStatusHelper.ts

```diff
@@ -172,4 +172,14 @@ export class CheckChatStatus {
 				return;
 			});
 	}
+
+	private async getAnalyticsQueueTime(read: IRead, assoc: RocketChatAssociationRecord) {
+		const { queueTimeStart } = await retrievePersistentData(read, assoc);
+		if (queueTimeStart === null) {
+			console.error(ErrorLogs.QUEUE_TIME_NOT_SET);
+			return '';
+		}
+		const dt = Math.ceil((Date.now() - queueTimeStart) / 1000);
+		return `${dt} seconds`;
```

Comment:
@AlexanderKanakis @ear-dev PR is working as expected. Just one doubt. Currently, it logs data as string `<t> seconds` i.e `10 seconds` , `999 seconds` etc.
Are we good with this or we should just pass numeric value? Just thinking  from the metrics perspective

Reply:
@Shailesh351 great question.... I just asked Molly and she would like an integer and not a string.  I assume we can pass integers in the topic message body?

Reply:
@Shailesh351 @ear-dev Fixed with new commits.